Configs/.config/hypr/hyprland.conf

```diff
@@ -31,6 +31,7 @@ exec-once = systemctl --user import-environment WAYLAND_DISPLAY XDG_CURRENT_DESK
 exec-once = $scrPath/polkitkdeauth.sh # Authentication dialogue for GUI apps
 exec-once = waybar # Launch the system bar
 exec-once = blueman-applet # Systray app for Bluetooth
+exec-once = udiskie & 
```

Comment:
this `&` is not required here.

Reply:
this was copied from hyperland docs , so i will remove it